### DISCUSSION TITLE
Sensors update interval

### DISCUSSION THREAD

**Discussion by andriuskr** [firstof9/openevse#46] [General]
Hello,

Where i can update interval that sensor.openevse_charging_current and sensor.openevse_current_power_usage sersors where updated every 5 seconds ?
This is needed for solar consumption automations.

**Comment by firstof9** [comment]
They are pushed from the OpenEVSE via websocket, they are not polled.

**Comment by TommySharpNZ** [comment]
Do we know what the default update interval is? Can it be changed in OpenEVSE?

**Reply by firstof9** [reply]
When a sensor updates, it's pushed.
Why would you want to not have the charger update data in real time?

**Reply by TommySharpNZ** [reply]
Awesome, yeah the more frequent the better!
